.gitignore

```diff
@@ -21,3 +21,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+/package-lock.json
```

Comment:
@SokoPaulSokool Good work so far however the `package.json` file should not be excluded.

Reply:
package-lock.json is always causing errors when I try to build the app. It has some babel package.

Reply:
I have removed it

Reply:
@SokoPaulSokool Go ahead and pick one of the two lock files and delete the other.